networking/network-verification.adoc

```diff
@@ -38,7 +38,7 @@ The network verification includes checks for each of the following requirements:
 * The VPC has `enableDnsSupport` enabled.
 * The VPC has `enableDnsHostnames` enabled.
 ifdef::openshift-dedicated[]
-* Egress is available to the required domain and port combinations that are specified in the xref:../osd_planning/aws-ccs.adoc#osd-aws-privatelink-firewall-prerequisites_aws-ccs[AWS firewall prerequisites] section.
+* Egress is available to the required domain and port combinations for either {product-title} (ROSA) or Google Cloud Platform. For ROSA the required domain and port combinations are specified in the xref:../osd_planning/aws-ccs.adoc#osd-aws-privatelink-firewall-prerequisites_aws-ccs[AWS firewall prerequisites] section. For GCP, they are specified in the xref:../osd_planning/gcp-ccs.adoc#ccs-gcp-customer-procedure_gcp-ccs[required customer procedure] section.
```

Comment:
@lihongan Can you please help to confirm with this part?

Reply:
@huiran0826 could you please help take a look at this? Thanks

Reply:
It seems to me not ovn egress feature, but cloud level?  Not sure.   @zhaozhanqi @anuragthehatter  any ideas?

Reply:
+1, it should be cloud level network

Reply:
@xueli181114 they are cloud provider network related, I'm not sure if @yunjiang29 and @jianli-wei could help review ?

Reply:
For what it's worth, the [Customer cloud procedures on GCP](https://75001--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_planning/gcp-ccs#ccs-gcp-customer-procedure_gcp-ccs) topic was written and approved in a previous Jira ticket. Please let me know if I need to separate the bullet points between AWS and GCP.